pkg/virt-handler/vm.go

```diff
@@ -222,6 +223,7 @@ func NewController(
 		vmiSourceInformer:           vmiSourceInformer,
 		vmiTargetInformer:           vmiTargetInformer,
 		domainInformer:              domainInformer,
+		cpuModelConfigMapInformer:   cpuModelConfigMapInformer,
```

Comment:
IMO we don't need the informer but the Indexer is enough, similar to https://github.com/kubevirt/kubevirt/pull/11379.
You can wait for the sync before start the vmController, in https://github.com/kubevirt/kubevirt/blob/70caad7899f6eee312fd7939743fc5947b289a56/cmd/virt-handler/virt-handler.go#L411

Reply:
Thanks for your note. I realized we don't need informer here. Instead, I found a object called [ConfigMapLister](https://github.com/kubernetes/client-go/blob/eea636f8f427d6d614ee1276e6ef21e54ef0d20b/listers/core/v1/configmap.go#L30) that seems to be specifically designed for this.
So, I updated the code to use it. What do you think?